### DISCUSSION TITLE
Issue with fare check configuration

### DISCUSSION THREAD

**Discussion by jdubb2** [jdholtz/auto-southwest-check-in#137] [General]
{ 
   "check_fares": true,
    "notification_urls": "mailto://user:pass@hotmail.com"

Using a config.json in docker container. Keep getting this error. 

Error in configuration file:
Expecting value: line 3 column 26 (char 52)

Appreciate any help

**Comment by jdholtz** [comment]
Do you have the ending }?

**Reply by jdholtz** [reply]
Hey @jdubb2, were you able to figure out this issue?
